modules/gitops-release-notes-1-14-2.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-2_{context}"]
+= Release notes for {gitops-title} 1.14.2
+
+{gitops-title} 1.14.2 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.2_{context}"]
+== Errata updates
+
+[id="RHBA-XXXX:NNNN-gitops-1-14-2-security-update-advisory_{context}"]
+=== RHBA-XXXX:NNNN - {gitops-title} 1.14.2 security update advisory
+
+Issued: 2024-11-11
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHBA-XXXX:NNNN[RHBA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-2_{context}"]
+== Fixed issues
+
+* Before this update, the New App panel behaved unexpectedly as it reopened immediately after being closed, and typing in fields causes the cursor and text to flash repeatedly. With this fix, the New App panel operates as intended. link:https://issues.redhat.com/browse/GITOPS-5788[GITOPS-5788]
```

Comment:
As mentioned in the slack thread https://redhat-internal.slack.com/archives/C06SPJE651R/p1731999243234959. We could remove this one

Reply:
Fixed, as per this comment I am removing this fixed issue from the content.